test/selenium/conf/selenium-test.conf

```diff
@@ -7,10 +7,12 @@ identity {
   test.users.secret = ${?IDENTITY_TEST_USERS_SECRET}
 }
 support.url = ${?SUPPORT_URL}
+contribution.url = ${?CONTRIBUTION_URL}
```

Comment:
👍 Don't forget to configure these in Travis!

Reply:
Done